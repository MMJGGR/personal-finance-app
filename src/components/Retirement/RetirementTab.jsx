import React from 'react'

export default function RetirementTab() {
  return <h2 className="text-2xl font-bold text-amber-700">Retirement</h2>
}
