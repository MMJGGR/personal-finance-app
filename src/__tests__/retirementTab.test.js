import React from 'react'
import { render } from '@testing-library/react'
import { FinanceProvider } from '../FinanceContext'
import RetirementTab from '../components/Retirement/RetirementTab'

beforeAll(() => {
  global.ResizeObserver = class { observe() {} unobserve() {} disconnect() {} }
})

test('retirement tab placeholder snapshot', () => {
  const { container } = render(
    <FinanceProvider>
      <RetirementTab />
    </FinanceProvider>
  )
  expect(container.firstChild).toMatchSnapshot()
})
