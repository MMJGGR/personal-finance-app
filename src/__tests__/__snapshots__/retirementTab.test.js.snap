// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`retirement tab placeholder snapshot 1`] = `
<h2
  class="text-2xl font-bold text-amber-700"
>
  Retirement
</h2>
`;
